edgexfoundry/go-mod-messaging
# fix: (jakarta) pass a copy of topic to redis subscriber

fixes #130 for Jakarta
Backport of #129

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-messaging/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Unit test suffice for concurrency issue
The follow verifies subscribe not broken for jakarta
- Use this branch in edgex-go (jakarta branch)
- Build core-data
- Start jakarta non-secure stack
- Stop core data container
- run core data from command line with DEBUG logging
- verify logs show core data receiving event from message bus

## New Dependency Instructions (If applicable)
